src/viam/components/power_sensor/power_sensor.py

```diff
@@ -90,3 +90,6 @@ def add_reading(name: str, reading, returntype: List) -> None:
         add_reading("power", pow, [float])
 
         return readings
+
+    async def get_geometries(self) -> List[Geometry]:
+        raise MethodNotImplementedError("GetGeometries").grpc_error
```

Comment:
thinking about this some more, a better move is to just not subclass Sensor - making it just unimplemented is a bit unwieldy, and there's no reason why it has to subclass Sensor anyway